cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaRoller.java

```diff
@@ -447,8 +443,8 @@ private void restartIfNecessary(NodeRef nodeRef, RestartContext restartContext)
                 LOGGER.debugCr(reconciliation, "Pod {} is now ready", nodeRef);
             }
         } catch (ForceableProblem e) {
-            if (isPodStuck(pod) || restartContext.backOff.done() || e.forceNow) {
-                if (canRoll(nodeRef, 60_000, TimeUnit.MILLISECONDS, true, restartContext)) {
+            if (restartContext.podStuck || restartContext.backOff.done() || e.forceNow) {
+                if (canRoll(nodeRef, 60, TimeUnit.SECONDS, true, restartContext)) {
```

Comment:
out of curiosity why are you changing from MILLISECONDS to SECONDS? I know it's the same value but AFAIK, the former is the one mostly used across the operator codebase.

Reply:
Tbh I think it's a personal preference, I am used to reason more in milliseconds than seconds so for me it's easier to make a mistake that these are 60 ms than 60 seconds. But as I said it's personal. Anyway I would be consistent with the rest of the codebase using milliseconds everywhere. We could discuss this and change in a different PR.

Reply:
I think we use MS through most of the code. So we should stick with it and not use seconds.